src/main/java/techreborn/items/tools/ItemAdvancedDrill.java

```diff
@@ -24,6 +24,7 @@
 
 package techreborn.items.tools;
 
+import codechicken.lib.raytracer.RayTracer;
```

Comment:
You cannot use classes from other mods.

Reply:
Feel free to copy a stripped down version of `RayTracer` from CCL as done in CoFH Core.

Reply:
Thanks, I think I will do this.

Reply:
What was wrong with my fix? I just tested it and it seemed to work ok?

Reply:
Nothing, this is a bad branch I am commiting from. Let me reset.